### PR TITLE
Port gif-pros to LVGL 8.3

### PR DESCRIPTION
**Pull request by AndrewLuGit**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,13 +17,13 @@ EXTRA_CFLAGS=
...
 
 # Set to 1 to enable hot/cold linking
-USE_PACKAGE:=0
+USE_PACKAGE:=1
 
 # Set this to 1 to add additional rules to compile your project as a PROS library template
 IS_LIBRARY:=1
 # TODO: CHANGE THIS!
 LIBNAME:=gif-pros
-VERSION:=2.1.1
```
Comment (theol0403) [inline Makefile:26]: That's not a logical semantic version bump. I'd say 3.0.0 is correct.

File: gif-pros@2.1.1.zip
Comment (theol0403) [inline gif-pros@2.1.1.zip:1]: Don't delete old artifacts, people can use it to install old versions.

**Review by theol0403 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (AndrewLuGit) [reply Makefile:26]: The rationale behind the version number is that the 8 stands for LVGL version 8. If that doesn’t make sense to you I’ll change it.

Comment (theol0403) [reply Makefile:26]: Yeah, usually libraries don't inherit version numbers from their dependencies.

**Review by theol0403 — APPROVED** [review]

**Review by theol0403 — COMMENTED** [review]
Can you add small note to the readme to download 2.1.1 if still using pros 3.

**Pull request merged**